### DISCUSSION TITLE
PrimeNG Documentation < 17

### DISCUSSION THREAD

**Discussion by djay1977** [primefaces/community#3301] [PrimeNG]
Hello,

I notice that the official PrimeNG website only offers documentation from v17 to next.
In our company, we have many projects below 17 and it's important to be able to access the documentation of older versions because for budgetary reasons, it's not possible to migrate these projects to recent versions of PrimeNG and Angular.

Would it be possible to publish on a new domain the documentation for previous versions in legacy.
Thanks

**Comment by denisko** [comment]
The documentation is in the github repository. You can clone the repository for your required branch and run the documentation locally.

**Reply by djay1977** [reply]
I just checked and there are no more branches lower than 17 on Github

**Reply by denisko** [reply]
There are still tags available, e.g.:
https://github.com/primefaces/primeng/tree/17.18.15

**Reply by denisko** [reply]
Or rather this one, if you want to go before release 17:
https://github.com/primefaces/primeng/tree/16.9.1

**Reply by djay1977** [reply]
Ah ok, thanks 👍

**Reply by RodrigoSaezCea** [reply]
Hello, I give you this URL where you can visit the old documentation from LTS v7 to v16.
Just replace the X (“primeng-vX-lts”) with the required version number in the url below.

https://www.primefaces.org/primeng-vX-lts

**Reply by danielbarcellos** [reply]
It doesn't work

**Comment by serdartaylan** [comment]
@djay1977 @denisko @RodrigoSaezCea 

https://primeng-v10.pages.dev/
https://primeng-v11.pages.dev/
https://primeng-v12.pages.dev/
https://primeng-v13.pages.dev/
https://primeng-v14.pages.dev/
https://primeng-v15.pages.dev/
https://primeng-v16.pages.dev/

**Reply by djay1977** [reply]
Possible to have also the documentation of version 5 to 9. I have old projects that run on these versions

**Reply by serdartaylan** [reply]
@djay1977 
https://primeng.pages.dev/

**Reply by Krishnaaugustine** [reply]
> @djay1977 @denisko @RodrigoSaezCea
> 
> https://primeng-v10.pages.dev/ https://primeng-v11.pages.dev/ https://primeng-v12.pages.dev/ https://primeng-v13.pages.dev/ https://primeng-v14.pages.dev/ https://primeng-v15.pages.dev/ https://primeng-v16.pages.dev/

great job mate

**Reply by djay1977** [reply]
Thank you very much for the work

**Reply by cmalpicag** [reply]
Thanks a lot
